hyper/http11/connection.py

```diff
@@ -150,6 +150,15 @@ def __init__(self, host, port=None, secure=None, ssl_context=None,
         #: the standard hyper parsing interface.
         self.parser = Parser()
 
+        # timeout
+        timeout = kwargs.get('timeout')
```

Comment:
`timeout` should be optional. I think it should be `None` - that means no timeout.